src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts

```diff
@@ -297,6 +417,11 @@ export class AddFilterWizardComponent implements OnInit {
           this.alertService.error(error.statusText);
         }
       });
+    if (this.installNewPlugin) {
```

Comment:
can you please add a comment in code, why there is timeout and moveNext on what conditions?